src/Symfony/Component/PropertyInfo/PropertyAttributesExtractorInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyInfo;
+
+/**
+ * @author Andrew Alyamovsky <andrew.alyamovsky@gmail.com>
+ */
+interface PropertyAttributesExtractorInterface
+{
+    /**
+     * Gets the attributes of the property.
+     *
+     * Returns an array of attributes, each attribute is an associative array with the following keys:
+     * - name: The fully-qualified class name of the attribute
+     * - arguments: An associative array of attribute arguments if present
```

Comment:
I would use `@param` annotation to describe these parameters

Reply:
Since the `@param` annotation is used for describing method's arguments, are you suggesting using it for the `string $class, string $property, array $context = []` arguments?